docs/using/installing/index.rst

```diff
@@ -4,17 +4,27 @@
 Installing Flocker
 ==================
 
-As a user of Flocker you will need to install the ``flocker-cli`` package which provides command line tools to control the cluster.
-This should be installed on a machine with SSH credentials to control the cluster nodes
-(e.g., if you use our Vagrant setup then the machine which is running Vagrant).
+The following diagram illustrates the architecture of the cluster you are about to install:
 
-There is also a ``clusterhq-flocker-node`` package which is installed on each node in the cluster.
-It contains the services that need to run on each node.
+.. image:: images/installintro.png
+   :width: 75%
+   :alt: Diagram illustrating the Flocker CLI and control service and agents.
 
-.. note:: The ``clusterhq-flocker-node`` package is pre-installed by the :ref:`Vagrant configuration in the tutorial <tutvagrant>`.
+The CLI is installed on your local machine via the ``flocker-cli`` package which provides command line tools to control the cluster. 
+This package also includes the ``flocker-ca`` tool, which you must use to generate certificates for all the Flocker components.
+
+The Flocker agents are installed on any number of hosts (or nodes) in the cluster where your containers will run.
+The agent software is included in the ``clusterhq-flocker-node`` package.
+
+There is also a Flocker control service which you must install on one of the agent hosts, or on a separate machine. 
+The control service is also included in the ``clusterhq-flocker-node`` package, but is activated separately later in these installation instructions.
+
+.. note:: The agents and control service are pre-installed by the :ref:`Vagrant configuration in the tutorial <tutvagrant>`.
```

Comment:
Not new but `tutvagrant` is not a good name. We are not that limited in characters and can use things like `vagrant-tutorial`. Elsewhere we have been using `-` between words.